package.json

```diff
@@ -96,7 +96,7 @@
     "execa": "7.1.1",
     "glob": "^10.3.3",
     "husky": "^8.0.3",
-    "jest": "29.6.4",
+    "jest": "29.7.0",
     "jest-diff": "^29.6.2",
     "jest-snapshot": "^29.6.2",
     "jest-specific-snapshot": "^8.0.0",
```

Comment:
HMMM - I notice that it bumped the jest version, but not the other packages.
Maybe we need to submit a bug report here?